src/status_im/contexts/wallet/send/select_asset/view.cljs

```diff
@@ -17,22 +17,21 @@
    {:id :tab/collectibles :label (i18n/label :t/collectibles) :accessibility-label :collectibles-tab}])
 
 (defn- asset-component
-  []
-  (fn [token _ _ {:keys [currency currency-symbol]}]
-    (let [on-press         #(rf/dispatch [:wallet/send-select-token
-                                          {:token    token
-                                           :stack-id :wallet-select-asset}])
-          token-units      (utils/total-token-units-in-all-chains token)
-          crypto-formatted (utils/get-standard-crypto-format token token-units)
-          fiat-value       (utils/total-token-fiat-value currency token)
-          fiat-formatted   (utils/get-standard-fiat-format crypto-formatted currency-symbol fiat-value)]
-      [quo/token-network
-       {:token       (:symbol token)
-        :label       (:name token)
-        :token-value (str crypto-formatted " " (:symbol token))
-        :fiat-value  fiat-formatted
-        :networks    (:networks token)
-        :on-press    on-press}])))
+  [token _ _ {:keys [currency currency-symbol]}]
```

Comment:
It was a form-2 component, but it wasn't needed. :eyes:  I've seen this pattern a few times in our code, let's try to fix it if we find it :smile: